switch-to-classicpress-from-wordpress.php

```diff
@@ -0,0 +1,106 @@
+<?php 
+/*
+Plugin Name:       Switch to ClassicPress from WordPress
+Plugin URI:        https://github.com/classicpress/classicpress-migration-plugin/
```

Comment:
Incorrect URL

Reply:
Why incorrect URL?

What is the correct URL?